src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -328,6 +328,9 @@ internal static Props ProxyProps(string typeName, ClusterShardingSettings settin
         /// </summary>
         protected IImmutableSet<Member> MembersByAge = ImmutableSortedSet<Member>.Empty.WithComparer(AgeOrdering);
 
+        // membersByAge contains members with these status
+        private static readonly HashSet<MemberStatus> MemberStatusOfInterest = new HashSet<MemberStatus> { MemberStatus.Up, MemberStatus.Leaving, MemberStatus.Exiting };
```

Comment:
If you want this to be read-only, change it to an `ImmutableHashSet` or some other non-modifiable object.